### ISSUE TITLE
allow binding terminal commands

### ISSUE THREAD

**Issue by yangkennyk** [Dygmalab/Bazecor#255]
**Is your feature request related to a problem? Please describe.**
I'd like to bind keys to execute terminal commands. For example connecting to a certain bluetooth device or any other sort of interaction via shell/terminal commands

**Describe the solution you'd like**
Adding another "shell script" tab to the macros section (input text, keys & delay, functions)

**Describe alternatives you've considered**
I'm currently binding key combinations in bazecore, then linking that combination via https://github.com/koekeishiya/skhd to execute shell commands

**Additional context**
I'm on macOS

**Comment by javierguzman** [comment]
Don't the macros fulfill your need?

**Comment by paul-esch** [comment]
> Don't the macros fulfill your need?

Nah, macros let me paste some text into a terminal window but that's certainly not the same as executing shell commands invisibly. Right now, I have a hotkey bound in Hammerspoon to move my mouse pointer to a predefined position `/opt/homebrew/bin/cliclick m:620,2500` but that is cumbersome and maintenance-unfriendly whereas it seems like such a simple thing to add to a Bazecor configured key.
